src/services/member/plugins/profile/schemas.ts

```diff
@@ -6,68 +6,85 @@ import { FastifySchema } from 'fastify';
 import { customType, registerSchemaAsRef } from '../../../../plugins/typebox';
 import { errorSchemaRef } from '../../../../schemas/global';
 
-const inputPublicProfileMemberSchemaRef = registerSchemaAsRef(
-  'publicProfile',
-  'Public Profile',
+const inputPublicProfileMemberSchemaRef = customType.StrictObject({
```

Comment:
This is not a ref anymore.
```suggestion
const inputPublicProfileMemberSchema = customType.StrictObject({
```